src/routes/file-centric-tsv/index.ts

```diff
@@ -61,13 +79,28 @@ const createDownloadHandler = ({
       esQuery,
       sortField: FILE_METADATA_FIELDS['object_id'],
     });
+
+    const columnsSchema =
+      columns &&
+      JSON.parse(decodeURIComponent(columns)).map(
```

Comment:
Looks like this works correctly. Do we want to move this code into its own method like `buildTsvSchemaFromColumnList(columns: Array<{header:string; getter:string;})`?